mesheryctl/internal/cli/root/system/config.go

```diff
@@ -204,13 +208,6 @@ mesheryctl system config eks
 		return nil
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-		eksCheck := exec.Command("aws", "--version")
-		eksCheck.Stdout = os.Stdout
-		eksCheck.Stderr = os.Stderr
-		err := eksCheck.Run()
-		if err != nil {
-			log.Fatalf("AWS CLI not found. Please install AWS CLI and try again. \nSee https://docs.aws.amazon.com/cli/latest/reference/ ")
-		}
```

Comment:
If we do need `aws` CLI because of usage in Line no 278 then we might be needing this again.

Have you tested this out @Althaf66? Exclusively without `aws` CLI tool